services/pub/pub-publisher.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  publisherId: Joi.string().allow(null).required(),
+}).required()
+
+class PubPublisher extends BaseJsonService {
+  static category = 'other'
+
+  static route = {
+    base: 'pub/publisher',
+    pattern: ':packageName',
+  }
+
+  static examples = [
+    {
+      title: 'Pub Publisher',
+      namedParams: { packageName: 'dart.dev' },
+      staticPreview: {
+        label: 'publisher',
+        message: 'dart.dev',
+        color: 'blue',
+      },
```

Comment:
This looks like the last remaining piece. Please update the example's `staticPreview` key to invoke the `render` function instead of keeping it hardcoded

Reply:
Ah - I missed this; it's now been updated to call `render()`.